Improvements to documentation (#870)
springernature/frontend-toolkits: toolkits/global/packages/global-status-message/README.md

</div>
```

#### Status changed

> If the status message changes, update the `message` and `type` properties. This means the screen reader will announce the new message. 

Reviewer: ```suggestion
If you change the message after the page has loaded, a screen reader would announce the new message.
```

(It's not expected the original data for the message will change in the browser, only on the server. Changes are more likely to happen with JavaScript.)

### Focus

Supply an `id` in the data to make the status message focusable programmatically.

> Focusing the message ensures that it is visible in the viewport and a screen reader will announce it.

Reviewer: I don't think it's clear, on reflection, when to focus and when to just rely on the live region. 

```suggestion
Focusing the message ensures it is visible in the viewport and a screen reader will announce it. If it is not appropriate to move focus to the message, rely on the live region for announcement in screen readers.
```

> The status message will likely be the best focus position for continued keyboard interaction.

Reviewer: ```suggestion
```
